libraries/abstractions/secure_sockets/lwip/iot_secure_sockets.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Amazon FreeRTOS Secure Sockets V1.1.6
+ * Amazon FreeRTOS Secure Sockets V1.1.7
```

Comment:
Do we keep the lwip and freertos+tcp sockets versions in unison always?  Not 100% sure if any changes were made to lwip